packages/sanity/src/structure/comments/src/components/list/CommentThreadLayout.tsx

```diff
@@ -45,6 +46,7 @@ interface CommentThreadLayoutProps {
   onNewThreadCreate: (payload: CommentCreatePayload) => void
   onPathSelect?: (nextPath: CommentsSelectedPath) => void
   readOnly?: boolean
+  mode: CommentsUIMode
```

Comment:
Nitpick: alphabetize props

Reply:
Updated, but I think we should aim to make it part of the lint rules if we want to keep it 100% consistent everywhere.